### PR TITLE
added undefined check in columns

### PR DESCRIPTION
**Pull request by manav1403**
**Description**

This PR fixes https://github.com/service-mesh-performance/service-mesh-performance/issues/248

**Notes for Reviewers**

The error when logged was

image

**identified Error**
The error was due to the presence of undefined columns in the data

**Correction**
Applying filter for undefined values
`let chartDataFiltered = chartData.filter(( element ) => element !== undefined);`

**Corrected Output**
image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/GrafanaCustomChart.js [hunk 1]
```diff
@@ -754,7 +758,7 @@ class GrafanaCustomChart extends Component {
       } else {
         mainChart = (
           <div>
-            <div className={classes.error}>{error && 'There was an error communicating with the server'}</div>
```
Comment (utkarsh-pro) [inline ui/components/GrafanaCustomChart.js:757]: Any reason to remove the error explicitly?

Comment (manav1403) [reply ui/components/GrafanaCustomChart.js:757]: I missed it its was for testing! 
Thanks

**Review by utkarsh-pro — DISMISSED** [review]
LGTM

**Review by leecalcote — APPROVED** [review]
Good stuff, @manav1403. Way to jump in and add value. 👍

**Comment by leecalcote** [comment]
Whoo-hoo, @manav1403 🎈

**Pull request merged**